tests/c/side-trace.c

```diff
@@ -21,35 +21,35 @@
 //     9
 //     10
 //     jit-state: deoptimise
-//     12
+//     ...
```

Comment:
Why have these become `...`?

Reply:
I will check gain

Reply:
This change is not needed any more.
Updated 👉  a82d1e2d9fa9b0cbefc2b496a975dcfabac6e15e